docs/guides/troubleshooting.md

```diff
@@ -89,6 +90,24 @@ a component.
 
 To fix this issue please make sure that all event listeners are cleaned up on dispose.
 
+## Problems with setup options
+
+If a player is inititalised without the expected setup options, it's usually because of using both the `data-setup` attriubte and the `videojs()` constructor. Take this example:
```

Comment:
```suggestion
If a player is inititalized without the expected setup options, it's usually because of using both the `data-setup` attribute and the `videojs()` constructor. Take this example:
```

Reply:
should probably be `initialized`